bcda/auth/client/oktaclient.go

```diff
@@ -177,7 +177,7 @@ func (oc *OktaClient) RequestAccessToken(creds Credentials) (OktaToken, error) {
 
 	req.Header.Add("Accept", "application/json")
 	req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
-	req.Header.Add("Cache-Control", "no-cache")
+	req.Header.Add("Cache-Control", "no-cache,must-revalidate")
```

Comment:
Is this setting the header on the request?  If I understand the finding correctly, I think we need to be setting it on the response.  This is what I see in an example response to `curl`:
```
< HTTP/2 200 
< content-type: application/json
< content-length: 1462
< expires: Tue, 10 Mar 2020 02:48:32 GMT
< cache-control: max-age=0, no-cache, no-store
< pragma: no-cache
< date: Tue, 10 Mar 2020 02:48:32 GMT
< strict-transport-security: max-age=86400
```

Reply:
Which endpoint are you curling to locally? Or are you pointing to the dev env? I get a 405 method locally.

```
curl -I http://localhost:3000/api/v1/swagger/
HTTP/1.1 405 Method Not Allowed
Connection: close
Date: Tue, 10 Mar 2020 13:02:49 GM
```

I believe that is calling L15-20 on `nocache.go` : 

```var noCacheHeaders = map[string]string{
	"Expires":         epoch,
	"Cache-Control":   "no-cache, no-store, no-transform, must-revalidate, private, max-age=0",
	"Pragma":          "no-cache",
	"X-Accel-Expires": "0",
}
```

As such, it is calling all the defined Cache-Control settings defined there and not being listed.

Reply:
This code change is in the `oktaclient`, which I'm not sure is being used. when the application is in `SSAS` auth mode.  😄 

This is what I was running to get a general sense of the response headers coming back (must be on VPN):
```
curl --insecure -v -L  -X GET http://test.bcda.cms.gov/api/v1/metadata
```